crates/bevy_ecs/src/system/commands/command_queue.rs

```diff
@@ -37,43 +42,55 @@ impl CommandQueue {
         /// SAFETY: This function is only every called when the `command` bytes is the associated
         /// [`Commands`] `T` type. Also this only reads the data via `read_unaligned` so unaligned
         /// accesses are safe.
-        unsafe fn write_command<T: Command>(command: *mut MaybeUninit<u8>, world: &mut World) {
+        unsafe fn write_command_and_get_size<T: Command>(
```

Comment:
I'd like a doc string here. In particular, I want to know what the usize returned is.

Reply:
I ended up consolidating the safety comments on the field that stores the function pointer. It seems like a more logical place for safety invariants.